kong/schema_service_test.go

```diff
@@ -0,0 +1,23 @@
+package kong
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGet(T *testing.T) {
+	assert := assert.New(T)
+
+	client, err := NewTestClient(nil, nil)
+	assert.Nil(err)
+	assert.NotNil(client)
+
+	entities := []string{"services", "routes", "targets", "upstreams", "plugins"}
```

Comment:
add all core entities, consult documentation or kong/db/schemas/entities dir in kong/kong repo

Reply:
Done, although I had to skip some entities, as `clustering_data_planes` and `parameters` are not present in < `2.3` and `workspaces` are only EE.

We should probably have a way to conditionally include or exclude test cases based on the running Kong version.

Reply:
> Done, although I had to skip some entities, as clustering_data_planes and parameters are not present in < 2.3 and workspaces are only EE.

No problem.

> We should probably have a way to conditionally include or exclude test cases based on the running Kong version.

We do. See runWhen* funcs.